src/crypto/hpke/aad.rs

```diff
@@ -0,0 +1,99 @@
+use std::fmt::{Display, Formatter};
+
+use super::{Epoch, KeyIdentifier};
+
+const SALT: &str = "private-attribution";
```

Comment:
This isn't really salt (salt is generally randomized), but a domain separation string.  That is, something that ensures that - if you use the same randomness for multiple application - you don't end up accidentally producing the same keys (which would be disastrous).